google-cloud-storage/src/test/java/com/google/cloud/storage/testing/RemoteStorageHelperTest.java

```diff
@@ -126,9 +128,12 @@ public String getNextPageToken() {
   public void testForceDelete() throws InterruptedException, ExecutionException {
     Storage storageMock = EasyMock.createMock(Storage.class);
     EasyMock.expect(blob1.getBlobId()).andReturn(BLOB_ID1);
-    EasyMock.expect(storageMock.delete(BLOB_ID1)).andReturn(true);
     EasyMock.expect(blob2.getBlobId()).andReturn(BLOB_ID2);
-    EasyMock.expect(storageMock.delete(BLOB_ID2)).andReturn(true);
+
```

Comment:
I just realized, can we just delete these tests? I think we're just writing code twice.